charts/crunchy/templates/PostgresCluster.yaml

```diff
@@ -49,9 +49,6 @@ spec:
           requests:
             cpu: {{ .Values.crunchy.pgmonitor.exporter.requests.cpu }}
             memory: {{ .Values.crunchy.pgmonitor.exporter.requests.memory }}
-          limits:
```

Comment:
Change to `limits: ~` to null them properly

Reply:
to clarify: limits only need to be null'd in the templates file. The values file is fine as-is.